src/google-fonts/index.js

```diff
@@ -89,129 +157,186 @@ function GoogleFonts() {
 	};
 
 	return (
-		<div className="wrap google-fonts-page">
-			<h1 className="wp-heading-inline">
-				{ __( 'Add Google fonts to your theme', 'create-block-theme' ) }
-			</h1>
-			<h3>
-				{ __(
-					'Add Google fonts assets and font face definitions to your currently active theme',
-					'create-block-theme'
-				) }{ ' ' }
-				({ theme?.name.rendered })
-			</h3>
-
-			{ ! googleFontsData?.items && (
-				<p>
-					<Spinner />
-					<span>
-						{ __(
-							'Loading Google fonts data…',
-							'create-block-theme'
-						) }
-					</span>
-				</p>
-			) }
-
-			{ googleFontsData?.items && (
-				<>
-					<div className="select-font">
-						<SelectControl
-							label={ __( 'Select Font', 'create-block-theme' ) }
-							name="google-font"
-							onChange={ handleSelectChange }
-						>
-							<option value={ null }>
-								{ __( 'Select a font…', 'create-block-theme' ) }
-							</option>
-							{ googleFontsData.items.map( ( font, index ) => (
-								<option key={ index } value={ index }>
-									{ font.family }
-								</option>
-							) ) }
-						</SelectControl>
-					</div>
-
-					<DemoTextInput />
-
-					{ selectedFont && (
-						<p>
+		<FontsPageLayout>
+			<main>
+				<h1 className="wp-heading-inline">
+					<BackButton />
```

Comment:
I'm not sure it's a good practice to place a button inside the h1, also it looks a bit strange. 

I think the position of the H1 should stay consistent from screen to screen (manage fonts -> add google fonts). Could the button instead be above the H1?

Reply:
Good suggestion. I moved the button out of the title and put it at the top of it.